### ISSUE TITLE
Twitter initials setting does not work

### ISSUE THREAD

**Issue by matthi4s** [zammad/zammad#1629]
### Infos:

* Used Zammad version: 2.1.x
* Used Zammad installation source: Ubuntu package
* Operating system: Ubuntu
* Browser + version: Chromium 62.0.3202.75

### Expected behavior:

* After turning off the twitter initials in the settings, the initials should not be appended anymore.

### Actual behavior:

* The initials are still appended.

### Steps to reproduce the behavior:

* Turn off twitter initials in the settings
* Open a twitter ticket (dm or tweet)
* Click on "reply" (You can already see the initials here)
* Write an answer
* Send the answer

I've looked in the code and found that the setting is checked in the [article_new.coffee](https://github.com/zammad/zammad/blob/develop/app/assets/javascripts/app/controllers/ticket_zoom/article_new.coffee#L140) and the `attributes` variable is set accordingly, but never used in the following code. I'm not sure if it's the correct way, but I managed to fix this by setting `articleTypes` `features` to be `attributes` for both direct message and twitter status types like this: 

````coffeescript      
    @articleTypes.push {
        name:              'twitter status'
        icon:              'twitter'
        attributes:        []
        internal:          false,
        features:          attributes
        maxTextLength:     280
        warningTextLength: 30
      }
````

**Comment by martini** [comment]
@muhammadn maybe this is already fixed in Zammad 2.3, see https://github.com/zammad/zammad/commit/09af88a578e56106ae160b8761ffbb24edcab920

**Comment by muhammadn** [comment]
verified working, so closing. Thanks @martini!
